packages/zitadel-client/package.json

```diff
@@ -9,46 +9,40 @@
   "exports": {
     ".": {
       "types": "./dist/index.d.ts",
-      "import": "./dist/index.js",
-      "require": "./dist/index.cjs"
+      "import": "./dist/index.js"
     },
     "./v1": {
       "types": "./dist/v1.d.ts",
-      "import": "./dist/v1.js",
-      "require": "./dist/v1.cjs"
+      "import": "./dist/v1.js"
     },
     "./v2": {
       "types": "./dist/v2.d.ts",
-      "import": "./dist/v2.js",
-      "require": "./dist/v2.cjs"
+      "import": "./dist/v2.js"
     },
     "./v3alpha": {
       "types": "./dist/v3alpha.d.ts",
-      "import": "./dist/v3alpha.js",
-      "require": "./dist/v3alpha.cjs"
+      "import": "./dist/v3alpha.js"
     },
     "./node": {
       "types": "./dist/node.d.ts",
-      "import": "./dist/node.js",
-      "require": "./dist/node.cjs"
+      "import": "./dist/node.js"
     },
     "./web": {
       "types": "./dist/web.d.ts",
-      "import": "./dist/web.js",
-      "require": "./dist/web.cjs"
+      "import": "./dist/web.js"
     }
   },
   "files": [
     "dist/**"
   ],
   "sideEffects": false,
   "scripts": {
-    "build": "tsup",
+    "build": "tsc -p tsconfig.prod.json",
```

Comment:
what is the main reason to use tsc instead of tsup?

Reply:
The default `tsconfig.json` would allow `tests` and `src` directories, while the production one only compiles the `src`.

- Why `tests` directory?

You could read a bit more here https://straw-hat-team.github.io/adr/adrs/0541531435/README.html